tab-command/src/service/main/list_tabs.rs

```diff
@@ -69,3 +93,26 @@ impl MainListTabsService {
         }
     }
 }
+
+fn is_active(tab: &WorkspaceTab, cwd: &str) -> bool {
+    let name = &tab.name;
+    if tab.doc == None && *name == get_working_tab() {
```

Comment:
I don't have a decent way of testing whether a tab comes from YAML.  It's possible for YAML tabs to have no docstring.  Anonymous tabs do have the `directory` property set properly at creation (though not updated).

I think it's fine to always check the directory.  Something like this:
```
return cwd.starts_with(tab.directory.as_path())
```

Reply:
I think I updated as requested!  Thanks for the mentoring!